src/xmpp/httpAuth.go

```diff
@@ -0,0 +1,17 @@
+package xmpp
+
+import (
+	"encoding/xml"
+)
+
+const (
+	NSHTTPAuth = "http://jabber.org/protocol/http-auth"
+)
+
+// XEP-0070: Verifying HTTP Requests via XMPP
+type Confirm struct {
+	XMLName xml.Name `xml:"http://jabber.org/protocol/http-auth confirm"`
+	Id      string   `xml:"id,attr"`
```

Comment:
This should be `ID` but it matches what I used already so let's leave it as `Id` for now and land this. However, I may fix the case of struct fields across the package - a breaking change, but worth it I think.